lib/context.mli

```diff
@@ -10,7 +10,7 @@ val fail : unit -> unit
 
 exception Context of string
 
-val init_context : float -> int * int -> bool -> unit
+val init_context : int * int * int * int -> float -> int * int -> bool -> unit
```

Comment:
We should have a named type for `int * int * int * int`, but that doesn't have to be part of this PR.

Reply:
This is an RGBA color which is only used in setting the background color. Which is why I was hesitant to give it an alias. I think our current color type should probably keep its name, any thoughts what this should be named?

Reply:
This conflict is why I thought it would be better to deal with it later. This is OK for now and shouldn't block the merge.

Q - can we have only one color type for both background/fill that has RGBA parts, or is that not supported by Cairo for strokes?

Reply:
My thought was that since the focus of this library is simplicity, making users deal with an alpha value(which is almost always going to be at 255/1.0) every time they want to set the color of a shape would be a nuisance. 

In my experience, creative coding libraries have a 'set_color' function that takes RGB and maybe another that takes RGBA or just alpha.

Cairo (and I believe HTML canvas) does support both